### DISCUSSION TITLE
On material data P_RAM and P_RAJ for fkm-nonlinear instead prediction from ultimate strength

### DISCUSSION THREAD

**Discussion by ThumserTKP** [boschresearch/pylife#47] [Ideas]
It would be benifical to enable the use of material curves based on own experimental data instead of the one based on ultimate strength according to the guideline. This is also decribed in the guideline, e.g. 2.5.3.1, 2.5.5.1and 2.8.5.1.

**Comment by johannes-mueller** [comment]
@maierbn maybe you want to comment.

**Comment by maierbn** [comment]
Yes, this would be a good addition. The user should be able to provide their own values for `P_RAM/RAJ,Z,WS`, `P_RAM/RAJ,D,WS`, `d`, `d1`, `d2`. 

The current workaround is to manually perform the steps in the Jupyter notebook [fkm_nonlinear_full.ipynb](https://github.com/boschresearch/pylife/blob/develop/demos/fkm_nonlinear/fkm_nonlinear_full.ipynb) or the script [assessment_nonlinear_standard.py](https://github.com/boschresearch/pylife/blob/develop/src/pylife/strength/fkm_nonlinear/assessment_nonlinear_standard.py). Instead of
```
assessment_parameters = parameter_calculations.calculate_material_woehler_parameters_P_RAM(assessment_parameters)
assessment_parameters = parameter_calculations.calculate_material_woehler_parameters_P_RAJ(assessment_parameters)
``` 
provide your own values in `assessment_parameters`.

**Reply by johannes-mueller** [reply]
Do you think it would make sense to do this for the 2.2.0 release or after it?

**Reply by maierbn** [reply]
I would do it afterwards. It also helps Kristina with her topic and would be a feature to be tested for itself.

**Reply by johannes-mueller** [reply]
Ok. Then I will reissue it after 2.2. 

@ThumserTKP, thanks for your input.
